cmd/hc/hc.go

```diff
@@ -441,7 +441,7 @@ func setupApp() (app *cli.App) {
 		if debug {
 			os.Setenv("DEBUG", "1")
 		}
-		holo.Initialize()
+		holo.Initialize(holo.InitializeProtocols)
```

Comment:
This makes it really easy for the app designer to start more protocols and doesn't really effect the api. I can do this with and optional argument, I just have to learn how to do it in go first.